lib/tasks/organisations.rake

```diff
@@ -23,4 +23,27 @@ namespace :organisations do
 
     puts "Created #{organisation.inspect}"
   end
+
+  desc "Output table with data about each organisation"
```

Comment:
I get a load of JSON objects rather than a table here - should I be seeing something different?

Reply:
A load of JSON objects is what we went with because it was easier to write the code for; would you prefer a different output format, or just changing the description?

Reply:
JSON makes sense I think, so changing the description should be fine. If it's not too much extra effort it might be helpful to output it as a JSON array rather than as separate objects but that's an optional suggestion

Reply:
I've updated the description.

Reply:
I didn't realise there was a name for this! Makes sense to leave it as-is then